src/constants.js

```diff
@@ -240,6 +240,20 @@ export const NANOCONTRACT_REGISTER_STATUS = {
   SUCCESSFUL: 'successful',
 };
 
+export const WALLETCONNECT_NEW_NANOCONTRACT_TX_STATUS = {
+  READY: 'ready',
+  FAILED: 'failed',
+  LOADING: 'loading',
+  SUCCESSFUL: 'successful',
+};
+
+export const NANOCONTRACT_BLUEPRINTINFO_STATUS = {
+  READY: 'ready',
+  FAILED: 'failed',
+  LOADING: 'loading',
+  SUCCESSFUL: 'successful',
+};
```

Comment:
I think you are right to raise this discussion and I believe it would fit a refactoring KTLO. I've opened this issue https://github.com/HathorNetwork/hathor-wallet-mobile/issues/522.